sirepo/auth_db/user.py

```diff
@@ -85,17 +85,25 @@ def is_expired(self, role):
         return r.expiration < sirepo.srtime.utc_now()
 
     def uids_of_paid_users(self):
+        import sqlalchemy.exc
+
         cls = self.__class__
-        return [
-            x[0]
-            for x in self.query()
-            .with_entities(cls.uid)
-            .filter(
-                cls.role.in_(sirepo.auth_role.PAID_USER_ROLES),
-            )
-            .distinct()
-            .all()
-        ]
+        for _ in range(5):
+            try:
+                return [
+                    x[0]
+                    for x in self.query()
+                    .with_entities(cls.uid)
+                    .filter(
+                        cls.role.in_(sirepo.auth_role.PAID_USER_ROLES),
+                    )
+                    .distinct()
+                    .all()
+                ]
+            except sqlalchemy.exc.OperationalError:
```

Comment:
Removed in d873c9e62. It's not necessary except in dev where the db is created at startup. I had fixed conftest.py (which will probably fix the spurious "cannot connect to job_supervisor" in GitHub Actions) to start the supervisor after the server, since the server creates the db. This code was here to try to allow the supervisor to startup asynchronously. It would be better if the supervisor did not touch auth_db and instead queried the server for the paid user list (#5595).